pkg/bazel/bazelisk.go

```diff
@@ -9,8 +9,14 @@
 package bazel
 
 import (
+	"aspect.build/cli/pkg/ioutils"
 	"bufio"
 	"fmt"
+	"github.com/bazelbuild/bazelisk/core"
```

Comment:
I guess your editor is auto-sorting these? we ought to setup pre-commit so it's the same for everyone

Reply:
Yeah when it saves IJ reordered these, and the pre-commit hooks didn't complain, so I guessed this was correct.